### PR TITLE
AzureRT PR - Update Tests & Split Large Code File

### PR DESCRIPTION
**Pull request by huangpf**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@huangpf__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (phuang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by yugangw-msft** [comment]
Lots of file, but does it include updating nuget version?

**Comment by huangpf** [comment]
@yugangw-msft Thanks for the comment. I shall update the version for ARM Compute, even though it's just splitting the files.

File: src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ListParameters.cs [hunk 1]
```diff
@@ -1,33 +1,39 @@
-﻿//
-// Copyright (c) Microsoft.  All rights reserved.
-//
+//
```
Comment (huangpf) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ListParameters.cs]: @yugangw-msft Why GitHub would think this is a file renaming change...

Comment (yugangw-msft) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ListParameters.cs]: guess the diff is not perfect

**Pull request merged**